### PR TITLE
Update Android build_tools_version and api_level in examples

### PR DESCRIPTION
**Pull request by Bencodes**
PRs are currently red as a result of this.

### CODE REVIEW

**Comment by jin** [comment]
Thanks for the PR! Could you remove the build_tools_version line altogether and rely on the autodiscovery mechanism in the rule?

File: examples/android_instrumentation_test/WORKSPACE [hunk 1]
```diff
@@ -3,7 +3,7 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 android_sdk_repository(
     name = "androidsdk",
     api_level = 28,
-    build_tools_version = "28.0.2",
+    build_tools_version = "30.0.3",
```
Comment (jin) [inline examples/android_instrumentation_test/WORKSPACE:6]: We can remove this version pinning altogether.

**Review by jin — CHANGES_REQUESTED** [review]

File: examples/android_instrumentation_test/WORKSPACE (hunk 1 above)
Comment (Bencodes) [reply examples/android_instrumentation_test/WORKSPACE]: Updated

**Review by jin — APPROVED** [review]
Thanks, and apologies for a long delay

**Pull request merged**